src/arrayMethodIndexOf.js

```diff
@@ -5,8 +5,22 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.indexOf2 = function(searchElement, fromIndex = 0) {
+    if (fromIndex < 0) {
+      for (let i = this.length + fromIndex; i > -1; i--) {
```

Comment:
Если индекс отрицательный - оно ищет от этого элемента до конца массива, все еще в прямом порядке, а не обратном
```suggestion
      for (let i = this.length + fromIndex; i < this.length; i++) {
```

Reply:
я пару раз перевіряв на код пені у мене працює , глянь ще раз ...

Reply:
я проверил в кодпене уже пару раз, вроде же все работает...

Reply:
![image](https://user-images.githubusercontent.com/38065883/69162871-f4e2f000-0af5-11ea-8252-44d5a60d97dc.png)
вот разница в работе

Reply:
Оригинальный индексОф находит элемент, потому что начинает с 'world' и идет до конца массива
Твой индексОф не находит элемент, потому что начинает с 'world' и идет в начало массива, и не встречает по пути pavlo